pennylane/templates/subroutines/qft.py

```diff
@@ -149,7 +150,9 @@ def compute_matrix(n_wires):  # pylint: disable=arguments-differ
         return np.fft.ifft(np.eye(2**n_wires), norm="ortho")
 
     @staticmethod
-    def compute_decomposition(wires, n_wires):  # pylint: disable=arguments-differ,unused-argument
+    def compute_decomposition(
+        wires: WiresLike, n_wires=None
```

Comment:
it's weird to me that `n_wires` exists at all 🤔

Reply:
Indeed. I'm trying a fix in the latest commit ... 🤞🏼

Reply:
The fix seems to work! 😄